lib/rules/no-magic-number.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * @fileoverview Rule to flag statements that use magic numbers (adapted from https://github.com/danielstjules/buddy.js)
+ * @author Vincent Lemeunier
```

Comment:
Don't forget to add your copyright here and in the tests. ([example](http://eslint.org/docs/developer-guide/working-with-rules))